sdk/eventgrid/eventgrid-systemevents/review/eventgrid-systemevents.api.md

```diff
@@ -1705,9 +1805,9 @@ export enum KnownRecordingFormatType {
 // @public
 export enum KnownServiceApiVersions {
     // (undocumented)
-    v2018_01_01 = "2018-01-01",
+    V20180101 = "2018-01-01",
```

Comment:
Is this a breaking change to completely change the API version like this?

Reply:
Could be a codegen change. @MaryGao any insights?  This package is still in preview.

Reply:
This is codegen change because we unified normalization style to PascalCase with enum name. Yes this is a breaking and our suggestion is for stable version we prefer to use clientName to mitigate this(//we have a fix for [stable ones ](https://github.com/Azure/autorest.typescript/issues/3107)here but if sth missing for data plane let me know we could have another fix) and for preview we prefer to directly take this change.